src/frontend/app/shared/data-sources/list-data-source.ts

```diff
@@ -220,9 +222,11 @@ export abstract class ListDataSource<T, A = T> extends DataSource<T> implements
   }
 
   getLocalPagesObservable(page$, pagination$: Observable<PaginationEntityState>, dataFunctions) {
-    return page$.pipe(
-      withLatestFrom(pagination$),
-      map(([entities, paginationEntity]) => {
+    return combineLatest(
+      pagination$,
```

Comment:
Shouldn't this stay withLatestFrom?

Reply:
We need to observe both the entities and pagination observables now (due to changes I made in the pagination entity observable) to ensure we get the new pages on page change.

Reply:
Ok, tested, got it